pom.xml

```
@@ -28,7 +28,7 @@
     <revision>1.23.2</revision>
     <changelist>-SNAPSHOT</changelist>
     <java.level>8</java.level>
-    <jenkins.version>2.150.3</jenkins.version> <!-- Should be kept in sync with the version resulting from Dockerfile FROM statement-->
+    <jenkins.version>2.190.1</jenkins.version> <!-- Should be kept in sync with the version resulting from Dockerfile FROM statement-->
```

nit: Can we delete this comment? It has been obsolete since https://github.com/jenkinsci/blueocean-plugin/commit/5cffd5346b628032ba52f74523f3a6dd7736c680 switched to using a floating tag as far as I understand, but every time I see the comment when the version is updated I go and check to see if it still matters. 

```suggestion
    <jenkins.version>2.190.1</jenkins.version>
```